kythe/proto/common.proto

```diff
@@ -89,6 +89,19 @@ message Diagnostic {
   string context_url = 4;
 }
 
+// A Diagnostic with its associated file.
+message ResolvedDiagnostic {
```

Comment:
I'm not sure what "resolved" here means. Can you talk a bit more about what resolution means in the comment?

Reply:
👍